src/Templates/DiffPlaces/createAllBadOpcodeTests.sh

```diff
@@ -28,18 +26,17 @@
 ./createBadOpcodeTest.sh 2D
 ./createBadOpcodeTest.sh 2E
 ./createBadOpcodeTest.sh 2F
-./createBadOpcodeTest.sh 49
-./createBadOpcodeTest.sh 4A
+./createBadOpcodeTest.sh 49 '<Cancun'
+./createBadOpcodeTest.sh 4A '<Cancun'
 ./createBadOpcodeTest.sh 4B
 ./createBadOpcodeTest.sh 4C
 ./createBadOpcodeTest.sh 4D
 ./createBadOpcodeTest.sh 4E
 ./createBadOpcodeTest.sh 4F
```

Comment:
it is still implicit that the script generates only >=London by default

Reply:
Yes, that's still the default. Should I change it to `>=Shanghai`?